packages/runner/src/setupServer.js

```diff
@@ -54,6 +54,9 @@ async function setupServer({ apiPort }) {
   });
   io.on('connection', wrapSocketErrors((socket) => {
     socket.emit('stateChanged', filterRoomState(roomState));
+    socket.on('ping', (cb) => cb({
```

Comment:
also remove the `get('/date')` endpoint further below in the same file. We can remove because the runner and runner frontend are versioned together